woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Block: add support for the on Sale Badge block

## What

This PR adds the support for On Sale Badge block to the Product Gallery block.

Fixes #10751 

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Visit the Single Product Template.
2. Add the Product Gallery Block.
3. Ensure that the Sale badge is visible.
4. Change the alignment of the Sale badge.
5. Visit a product in sale and ensure the On Sale Badge is visible and in the correct position.
6. Visit a product not in sale and ensure that the On Sale Badge isn't visible.


## Screenshots or screencast





https://github.com/woocommerce/woocommerce-blocks/assets/4463174/6c9f490e-a5b8-4cdd-b680-539520d19129




## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
